app/controllers/api/v1/events_controller.rb

```diff
@@ -51,6 +51,25 @@ def show
         )
       end
 
+      def estimate_fees
+        result = Fees::EstimateInstantService.call(
+          organization: current_organization,
+          params: create_params,
+        )
+
+        if result.success?
+          render(
+            json: ::CollectionSerializer.new(
+              result.fees,
+              ::V1::FeeSerializer,
+              collection_name: 'fees',
+            )
```

Comment:
Style/TrailingCommaInArguments: Put a comma after the last parameter of a multiline method call. (https://rubystyle.guide#no-trailing-params-comma)